content/en/docs/tasks/administer-cluster/running-cloud-controller.md

```diff
@@ -53,6 +53,12 @@ Successfully running cloud-controller-manager requires some changes to your clus
   kubelet is aware that it must be initialized by the cloud controller manager
   before it is scheduled any work.
 
+  {{< note >}}
+  `--cloud-provider` flag usage varries according to the user. If the user has
+  external CCMs(not the internal cloud controller loops in the Kubernetes Controller
```

Comment:
```suggestion
  external CCMs (not the internal cloud controller loops in the Kubernetes Controller
```
a nit

Reply:
@windsonsea,
Thanks for noticing, i have updated it.